src/components/layout/besquare/style/footer.js

```diff
@@ -66,9 +65,9 @@ export const ImageWrapper = styled(QueryImage)`
     }
 `
 
-export const TextContainer = styled(Flex)`
+export const TextContainer = styled.div`
```

Comment:
Nit: Could we potentially use `CssGrid` for this component instead?

Reply:
It breaks up the layout on safari 😅